src/niri.rs

```diff
@@ -386,6 +391,47 @@ impl State {
         self.niri.queue_redraw_all();
     }
 
+    // moves cursor within the specified rectangle, only adjusting coordinates if needed
+    fn move_cursor_within_rect(&mut self, rect: Rectangle<f64, Logical>, mode: CenteringMode) {
```

Comment:
```suggestion
    fn move_cursor_to_rect(&mut self, rect: Rectangle<f64, Logical>, mode: CenteringMode) {
```

Reply:
fixed in commit [123c2f0](https://github.com/YaLTeR/niri/pull/245/commits/123c2f040e1891c7ce5d3d7c29d497f48da788cd)